cli/azd/pkg/tools/kubectl/kubectl.go

```diff
@@ -184,7 +193,7 @@ func (cli *kubectlCli) ConfigView(
 	return &res, nil
 }
 
-func (cli *kubectlCli) ApplyWithInput(ctx context.Context, input string, flags *KubeCliFlags) (*exec.RunResult, error) {
+func (cli *kubectlCli) ApplyWithStdIn(ctx context.Context, input string, flags *KubeCliFlags) (*exec.RunResult, error) {
```

Comment:
StdIn sounds like an io.Reader, but this is just taking a string.  Can you use a different name to avoid confusion with the commandRunner names;

Reply:
Optionally, change input from string to io.Reader

Reply:
I'd like to keep the contract for our wrapping service to be simple and just accept a simple `string`.  Internally this value is converted into an `io.Reader` and when executed via command runner sends value with `stdin` to `kubectl`.